proxies/live/apiproxy/resources/jsc/ConnectionMetadata.SetRequestHeaders.js

```diff
@@ -50,7 +50,10 @@
   }
 
   var nrlAppID = context.getVariable("app.nrl-app-id");
```

Comment:
I'm guessing this doesn't fail if there is no variable and it only retrurns null or empty?

Reply:
yeah its like a .get on a dictionary in python if it's not there it returns null